tools/llvm-cbe/llvm-cbe.cpp

```diff
@@ -331,13 +331,12 @@ static int compileModule(char **argv, LLVMContext &Context) {
   Options.UnsafeFPMath = codegen::getEnableUnsafeFPMath();
   Options.NoInfsFPMath = codegen::getEnableNoInfsFPMath();
   Options.NoNaNsFPMath = codegen::getEnableNoNaNsFPMath();
-  Options.HonorSignDependentRoundingFPMathOption =
-      codegen::getEnableHonorSignDependentRoundingFPMath();
+  Options.HonorSignDependentRoundingFPMathOption =codegen::getEnableHonorSignDependentRoundingFPMath();
   if (codegen::getFloatABIForCalls() != FloatABI::Default)
     Options.FloatABIType = codegen::getFloatABIForCalls();
   Options.NoZerosInBSS = codegen::getDontPlaceZerosInBSS();
   Options.GuaranteedTailCallOpt = codegen::getEnableGuaranteedTailCallOpt();
-  Options.StackAlignmentOverride = codegen::getOverrideStackAlignment();
+  //Options.StackAlignmentOverride = codegen::getOverrideStackAlignment();
```

Comment:
Also seems hacky:

Reply:
for this one i was unable to find a replacement in the options